src/renderer/ui/utils/constants.js

```diff
@@ -42,9 +42,10 @@ export const ACTION_KEYS = _.transform(_.range(26), (final, current) => { // let
 })));
 
 export const languageMap = {
-  de: 'German',
-  'en-US': 'English',
+  cs: 'Czech',
   'nl-NL': 'Dutch',
```

Comment:
My standards say that the object should be sorted alphabetically by **key** not value.  The original order was correct, please put `en-US` and `de` above `nl-NL`